packages/wonder-blocks-cell/src/components/internal/cell-core.tsx

```diff
@@ -127,7 +128,7 @@ function CellInner(props: CellCoreProps): React.ReactElement {
             />
 
             {/* Cell contents */}
-            <View style={styles.content} testId={testId}>
+            <View style={[styles.content, contentStyle]} testId={testId}>
```

Comment:
**suggestion:** It would be helpful adding a unit test for this case. For reference: https://github.com/Khan/wonder-blocks/blob/860d9ef9358a4bf5b8c27126c2e0d047d62a7642/packages/wonder-blocks-cell/src/components/internal/__tests__/cell-core.test.tsx#L128-L140